bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs

```diff
@@ -185,17 +201,48 @@ impl BackupRecoveryKey {
     ) -> Result<String, PkDecryptionError> {
         self.inner.decrypt_v1(&ephemeral_key, &mac, &ciphertext).map_err(|e| e.into())
     }
+
+    /// Try to decrypt a message that was encrypted using the public part of the
+    /// backup key.
+    pub fn decrypt_session_data(
+        &self,
+        session_data: EncryptedSessionData,
+    ) -> Result<BackedUpRoomKey, PkDecryptionError> {
```

Comment:
The compiler complains here about:

```
error[E0277]: the trait bound `BackedUpRoomKey: LowerReturn<UniFfiTag>` is not satisfied
  --> bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs:88:1
   |
88 | #[uniffi::export]
   | ^^^^^^^^^^^^^^^^^ the trait `LowerReturn<UniFfiTag>` is not implemented for `BackedUpRoomKey`
   |
   = help: the following other types implement trait `LowerReturn<UT>`:
             <bool as LowerReturn<UT>>
             <i8 as LowerReturn<UT>>
             <i16 as LowerReturn<UT>>
             <i32 as LowerReturn<UT>>
             <i64 as LowerReturn<UT>>
             <u8 as LowerReturn<UT>>
             <u16 as LowerReturn<UT>>
             <u32 as LowerReturn<UT>>
           and 59 others
   = note: required for `Result<BackedUpRoomKey, PkDecryptionError>` to implement `LowerReturn<UniFfiTag>`
   = note: this error originates in the attribute macro `uniffi::export` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `BackedUpRoomKey: LowerReturn<UniFfiTag>` is not satisfied
   --> bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs:210:10
    |
210 |     ) -> Result<BackedUpRoomKey, PkDecryptionError> {
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `LowerReturn<UniFfiTag>` is not implemented for `BackedUpRoomKey`
    |
    = help: the following other types implement trait `LowerReturn<UT>`:
              <bool as LowerReturn<UT>>
              <i8 as LowerReturn<UT>>
              <i16 as LowerReturn<UT>>
              <i32 as LowerReturn<UT>>
              <i64 as LowerReturn<UT>>
              <u8 as LowerReturn<UT>>
              <u16 as LowerReturn<UT>>
              <u32 as LowerReturn<UT>>
            and 59 others
    = note: required for `Result<BackedUpRoomKey, PkDecryptionError>` to implement `LowerReturn<UniFfiTag>`
```

I'm not sure what's the best way to fix it.  Does `BackedUpRoomKey` need to implement some trait?  Or get wrapped in something?